test/test.py

```diff
@@ -0,0 +1,98 @@
+import munge
```

Comment:
tired: unsorted imports
wired: sorted alphabetically (what you have here)
inspired: three blocks of imports, each sorted alphabetically - built-in packages (e.g. collections, sys, unittest), installed packages (e.g. gensim, numpy), and local files (e.g. munge)

Reply:
While we're at it, guidance on spacing (which I think you mostly follow):

between big things (e.g. classes, functions outside classes): two blank lines
between small things (e.g. functions in a class, logically united chunks of code within a function): one blank line
between function and doctring or between docstring and start of code: no blank lines

Reply:
i like the 'sassy' comments for the record haha